### ISSUE TITLE
[Quest][Desolace] Return to Vahlarriel - escort not beginning

### ISSUE THREAD

**Issue by tcberk12** [Atlantiss/NetherwingBugtracker#1552]
**Description**: 

I just started this escort quest (http://classicdb.ch/?quest=1440) inside Thunder Axe Fortress. Upon accepting, Dalinda Malem instantly aggroed to a mob, killed it, then returned to her spawn point. After this, she did not move from that spot despite the fact that the escort had started. After waiting a few minutes, I abandoned the quest, but I was unable to pick up off the quest off of her again.

**Current behaviour**:

Dalinda does not move once starting the escort. My initial guess is that she would have begun moving as intended if she had not aggroed onto a mob instantly upon accepting the quest (it had just respawned as I accepted the quest).

Even after abandoning, she is permanently reacting aggressively to new spawns, then returning to her spawn point, and will not offer the quest again. She should just be passive towards the mobs in the fortress until you start the escort.

<img width="1438" alt="screen shot 2018-10-12 at 4 46 06 pm" src="https://user-images.githubusercontent.com/43530009/46898635-b9301e00-ce3f-11e8-86f9-a4d0eb507718.png">

In the SS you can see in the chatlog I've begun the quest, but Dalinda is still in her original spot. The mob that she instantly aggroed onto, killed, and presumably caused this problem is in the bottom left. Ignore the enemy name plate, that's a mob that I just caught right as its spawning (appeared visually like .1s later) 

**Expected behaviour**: 

Dalinda Malem should walk out of Thunder Axe Fortress once you begin the escort. Shouldn't matter if she aggroes onto a mob right away, she should just then return to the start spot and begin walking out.

**Server Revision**: 
2169

**Comment by tcberk12** [comment]
UPDATE: I returned an hour later and the quest was available. I made sure to clear every mob in the area so that she would not aggro onto anything immediately, and she began her path as intended once I started it.

That must mean the issue was caused by the escort npc aggroing onto a mob as soon as the quest started/she became aggressive to the surrounding mobs. For whatever reason, this caused her to not begin her escort pathing after killing the initial mob.
